pkg/apis/application/v1alpha1/types.go

```diff
@@ -2633,11 +2634,13 @@ type KustomizeOptions struct {
 type ApplicationDestinationServiceAccount struct {
 	// Server specifies the URL of the target cluster's Kubernetes control plane API. This must be set if Name is not set.
 	Server string `json:"server,omitempty" protobuf:"bytes,1,opt,name=server"`
+	// Name is an alternate way of specifying the target cluster by its symbolic name. This must be set if Server is not set.
+	Name string `json:"name,omitempty" protobuf:"bytes,2,opt,name=name"`
```

Comment:
`Name` is not explicitly supported. User has to provide `Server` URL in this config